web/app/css/tap.css

```diff
@@ -18,3 +18,20 @@
     color: rgba(52, 137, 206, 0.906);
   }
 }
+
+.tap-more-info {
+  padding-right: 50px;
+
+  & .tap-info-section {
+    padding: 24px 0;
+
+    & hr {
+      color: red;
+
```

Comment:
Nit: Extra space

Reply:
oooops. also that `color: red` should not be in here 😂

Reply:
good catch! 😂 I didn't even realize that.